src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -245,6 +277,67 @@ def migrate_layers():
             'slugified_time_regex_format': slugified_time_regex_format,
         }
 
+        # Move input files
```

Comment:
I suggest to delete `input_chunk` directory before input files using `shutil.rmtree`

Reply:
Incorporated.